_docs/addons/yform/yf_datatable_XofY.md

```diff
@@ -43,22 +43,21 @@ YForm-EP vor der Tabellenausgabe angezogen wird.
 
 ```php
 \rex_extension::register('YFORM_DATA_LIST',
-    function( \rex_extension_point $ep )
-    {
+    static function (rex_extension_point $ep) {
         // Anzahl Zeilen in der Query
-        $lines = $ep->getSubject()->getRows();
+        $lines = $ep->getSubject()->getRows(); /** @phpstan-ignore-line */
 
         // Anzahl Zeilen in der Tabelle
         $sql = \rex_sql::factory();
-        $sql->setQuery('select count(*) FROM '.$ep->getParams()['table']['table_name']);
+        $sql->setTable($ep->getParams()['table']['table_name']);
+        $sql->select('count(*)');
         $total = $sql->getValue('count(*)');
 
         // Tabellentitel nur erweitern wenn gefiltert.
-        if( $lines < $total )
-        {
-            \rex_i18n::addMsg( 'yform_tabledata_overview', "Datentabelle ($lines von $total)" );
+        if ($lines < $total) {
+            \rex_i18n::addMsg('yform_tabledata_overview', 'Datentabelle (' . $lines . ' von '. $total . ')');
```

Comment:
Hat der Fixer das umgebaut? Wenn nein: was stört an der einfacher zu lesenden vorherigen Fassung?

Reply:
Das ist bei mir wohl schon ein Automatismus alles von " auf ' umzustellen :)

Reply:
Meine aktuelle Marotte ist `sprintf('Datentabelle (%d von %d)',$lines,$total)`

Reply:
Ich ersetze auch in JavaScript alle " durch ' 

Bei Loops über mehrere tausend Zeilen hat das auch durchaus Performance-Auswirkungen.

An der Stelle aber unnötig. Ändere ich wieder zurück